tests/installation/bootloader_zkvm.pm

```diff
@@ -68,6 +68,9 @@ sub set_svirt_domain_elements {
 sub run {
     my $svirt = select_console('svirt', await_console => 0);
 
+    $svirt->get_cmd_output('free -h');
```

Comment:
I would use record info and combine the three variables. And maybe modify the post_fail_hook to get these stats if the job finished as incomplete, but that would be a separate ticket I guess...